packages/grafana-ui/src/components/Card/Card.tsx

```diff
@@ -263,6 +262,7 @@ export const getCardStyles = stylesFactory((theme: GrafanaTheme) => {
     innerLink: css`
       display: flex;
       width: 100%;
+      padding: ${theme.spacing.md};
```

Comment:
Inner link is an optional component, so the Card won't have any padding now if it's not present: 
![Screenshot 2021-04-06 at 9 40 48](https://user-images.githubusercontent.com/8878045/113669308-3d378980-96bc-11eb-80ea-cd75b4f25ba2.png)

Reply:
hmm...good point.
That means we need to somehow add back the padding to the Card component.

Any ideas on the best possible solution for this? @Clarity-89

Reply:
I've made a fix for this here: https://github.com/grafana/grafana/pull/32702

Reply:
oh wow...Thanks alot! @Clarity-89 
Your solution is quite simple and elegant; love it!